backend/app/Models/BatchStep.php

```
@@ -106,6 +106,72 @@ public function lotConsumptions(): HasMany
+     * this batch step yet - the items that block completion. Empty when none.
+     *
+     * @return \Illuminate\Support\Collection<int, string>
+     */
+    public function pendingRequiredChecklistLabels(): \Illuminate\Support\Collection
+    {
+        $templateId = $this->batch?->workOrder?->process_snapshot['template_id'] ?? null;
+        if (! $templateId) {
+            return collect();
+        }
+
+        $required = TemplateStepChecklistItem::where('process_template_id', $templateId)
```

PHPStan (2.2.2)

[error] Call to an undefined static method App\Models\TemplateStepChecklistItem::where().


(staticMethod.notFound)